components/elm/src/biogeochem/SatellitePhenologyMod.F90

```diff
@@ -371,7 +375,7 @@ subroutine SatellitePhenology(bounds, num_filter, filter, &
          ! snow burial fraction for short vegetation (e.g. grasses) as in
          ! Wang and Zeng, 2007.
 
-         if (veg_pp%itype(p) > noveg .and. veg_pp%itype(p) <= nbrdlf_dcd_brl_shrub ) then
+         if (veg_pp%itype(p) > noveg .and. woody(veg_pp%itype(p)) == 1.0_r8 ) then
```

Comment:
Double check this: Should be shrubs, or shrubs + grasses?

Reply:
nbrdlf_dcd_brl_shrub: broadleaf_deciduous_boreal_shrub, the last shrub type index. <= implies all trees and shrubs. So should be edited as 'woody(veg_pp%itype(p)) >= 1.0_r8